### ISSUE TITLE
Jest - a11y tests call console.error

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#6224]
## 🐛 Bug

The _web_ app has three a11y tests that begin to fail when Jest is set to fail on console.error.

The _web_ package tests should not be calling console.error or it should be handled if expected

## 🕵️ Details

```
Default Client: No client has been specified using urql's Provider.
Warning: Encountered two children with the same key, `test-file-stub`.
```

## 📋 Steps to Reproduce

1. (optional) Check out branch `tooling_5846_jest-notify` from #6178
2. Run `npx jest --testNamePattern=accessibility` from the `apps/web` directory
3. Observe three tests failing

## 📸 Screenshot
![image](https://user-images.githubusercontent.com/8978655/230398740-f62882c3-b20f-4eba-8c6f-5f12f3dcd814.png)

## ✅ Acceptance Criteria

- [ ] Running the a11y tests results in no errors
- [ ] Tests continue to function as intended

**Comment by petertgiles** [comment]
Note to self:
> apps/web$ npx jest src/pages/DirectivePage/DirectivePage.test.tsx  --testNamePattern=accessibility
